docs/development/testguide.rst

```diff
@@ -67,50 +69,65 @@ options require the `pytest-pep8 plugin
 <https://pypi.org/project/pytest-pep8>`_, which must be installed
 separately.
 
-setup.py test
-=============
+.. _running-pytest:
+
+pytest
+======
+
+The test suite can be run directly from the native ``pytest`` command. In this
+case, it is important for developers to be aware that they must manually
+rebuild any extensions by running::
+
+    python setup.py build_ext --inplace
 
-The astropy core package and the Astropy package template provide a ``test``
-setup command, invoked by running ``python setup.py test`` while in the
-package root directory. Run ``python setup.py test --help`` to see the
-options to the test command.
+before running the test with pytest with::
 
-Since ``python setup.py test`` wraps the widely-used pytest framework, you may
-from time to time want to pass options to the ``pytest`` command itself. For
-example, the ``-x`` option to stop after the first failure can be passed
-through with the ``--args`` argument::
+    pytest
 
-    > python setup.py test --args "-x"
+Instead of calling ``build_ext --inplace``, you can also install the package
+in editable mode::
 
-`pytest`_ will look for files that
-:ref:`look like tests <pytest:python test discovery>`, per default
-in the current directory and all recursive directories, then run all
-the code that looks like tests (essentially all functions or methods
-with ``test_`` prefixes or inside ``Test`` prefixed classes) within
-those files.
+    pip install -e .[test]
 
-Turn on PEP8 checking by passing ``--pep8`` to the ``test`` command. This will
-turn off regular testing and enable PEP8 testing.
+The latter is required if you need to test parts of the package that rely
+on certain entry points being installed.
 
-Note also that this test runner actually installs astropy into a temporary
-directory and uses that for running the tests.  This means that tests of things
-like entry points or data file paths should act just like they would once
-astropy is installed.  The other two approaches described below do *not* do
-this, and hence may give different results when run from the astropy source
-code. Hence if you're running the tests because you've modified code that might
-be impacted by this, the ``setup.py test`` approach is the recommended method.
+It is possible to run only the tests for a particular subpackage or set of
+subpackages.  For example, to run only the ``wcs`` tests from the
+commandline::
+
+    pytest -P wcs
```

Comment:
If this is possible and does do `docs` too, then use it in circle-ci!!

Reply:
Done!